test/server.cpp

```diff
@@ -77,7 +77,7 @@ const ResourceCollection resources200Compressible{
   { DYNAMIC_CONTENT, "/ROOT%23%3F/skin/taskbar.css" },
   { STATIC_CONTENT,  "/ROOT%23%3F/skin/taskbar.css?cacheid=42e90cb9" },
   { DYNAMIC_CONTENT, "/ROOT%23%3F/skin/viewer.js" },
-  { STATIC_CONTENT,  "/ROOT%23%3F/skin/viewer.js?cacheid=00e0fdf3" },
+  { STATIC_CONTENT,  "/ROOT%23%3F/skin/viewer.js?cacheid=8c3370b7" },
```

Comment:
Regarding the commit message - it reads "Frontend: prevent empty search queries (update viewer.js cacheid test)". There is no need to include the note on cacheids in the message title/summary. If you want to preserve it, you can move it to the body of the message where more elaborate description and  additional details should belong.